scripts/xapi.service

```diff
@@ -1,6 +1,21 @@
 [Unit]
 Description=XenAPI server (XAPI)
-After=syslog.target forkexecd.service xcp-networkd.service message-switch.service v6d.service save-boot-info.service attach-static-vdis.service genptoken.service xcp-rrdd.service xapi-storage-script.service xenopsd-xc.service xenstored.service xcp-networkd.service
```

Comment:
Does systemd rearrange the requirements anyway to make it safe to reorder the services in the `Atfer=` or we need to preserve it for any reason?